tests/stdlib/tunittestpass.nim

```diff
@@ -0,0 +1,19 @@
+discard """
+  targets: "c js"
+"""
+
```

Comment:
@xflywind why not in tests/stdlib/tunittest.nim ?

Reply:
`tests/stdlib/tunittest.nim ` is very messy.